packages/tailwind-constructor/tailwind.config.js

```diff
@@ -24,13 +24,19 @@ module.exports = withTV({
     },
     data: {
       checked: 'state="checked"', //data-[state=checked]
+      open: 'state="open"', //data-[state=open]
     },
     fontFamily: filterTokensByType('fontFamilies', tokens),
     spacing: filterTokensByType('spacing', tokens),
     screens: {
       'sm': '640px',
       'md': '768px',
       'lg': '1024px',
+    },
+    extend: {
+      minWidth: {
+        128: "512px",
```

Comment:
why are we having this new value? we should stick to what the design team give us

Reply:
Any suggestions then on creating a min-width for the 448px? I used 512px because it was double of min-w-64 (256px). and didn't want to use [].

Reply:
how did you come up with these? `max-md:w-11/12 md:min-w-128`. mind telling me the process of extracting them from the

Reply:
at small screens I wanted the modal took up most of width of the screen, that's why I did 11/12 width. I realize I also have mx-2 or something too. 

md:min-w-128, was that I thought 128 or 512px was going to be a common use size, something we can extend tailwind for. w-64 exists, so I was just thinking double of that would be something we would want more in the future.  Just so the modal always have a minimum size, which I grabbed from figma.

Reply:
![image](https://github.com/dev-launchers/dev-launchers-platform/assets/109195898/d9305287-8852-4b74-b1af-86475a46f6de)

Reply:
@Nouri-Anouar do you want to add this new value to tokens studio?